browser/brave_browser_process_impl.cc

```diff
@@ -80,6 +79,7 @@
 #if defined(OS_ANDROID)
 #include "chrome/browser/flags/android/chrome_feature_list.h"
 #else
+#include "brave/browser/ui/brave_browser_command_controller.h"
```

Comment:
`//brave/browser/ui/brave_browser_command_controller.h` should be included in non-android build.